src/modules/edc-demo/components/transfer-history/transfer-history-viewer.component.html

```diff
@@ -1,18 +1,39 @@
 <div id="wrapper">
-  <div *ngIf="transferProcesses$ | async as transferProcesses">
-    <button mat-raised-button color="primary" (click)="loadTransferProcesses()">
-      <mat-icon>cached</mat-icon>
-      Refresh
-    </button>
-    <mat-paginator
-      pageSize="transferProcesses.length"
-      hidePageSize="true"
-      [length]="transferProcesses.length"></mat-paginator>
-
+  <button mat-raised-button color="primary" (click)="loadTransferProcesses()">
+    <mat-icon>cached</mat-icon>
+    Refresh
+  </button>
+  <mat-paginator
+    *ngIf="transferProcessesList.isReady"
+    pageSize="transferProcessesList.data.transferProcesses.length"
+    hidePageSize="true"
+    [length]="
+      transferProcessesList.data.transferProcesses.length
+    "></mat-paginator>
+  <div class="flex flex-row justify-center uppercase">
```

Comment:
The uppercase class here is right now making empty messages uppercase.

It should be removed here and empty state / error state should specify such styling.